intfloat/e5-mistral-7b-instruct model in sentence-transformer

fzowl opened the pull request.

fzowl committed:
Small corrections

fzowl committed:
intfloat/e5-mistral-7b-instruct model in sentence-transformer

fzliu commented on ebr/models/sentence_transformers.py:
```diff
@@ -31,6 +31,17 @@ def embed(self, data: str, input_type: str) -> list[list[float]]:
+    embd_dtype="float32",
+    embd_dim=4096,
+    num_params=22_700_000,
+    max_tokens=256,
```
Both of the above parameters need to be updated.

fzowl replied:
I removed these params as i did not find the correct values: https://huggingface.co/intfloat/e5-mistral-7b-instruct

fzowl committed:
Google text embedding models (#4)

* Google text embedding models

* Google text embedding models

* Google text embedding models

fzliu committed:
Fix model inference bugs, add more OpenAI models

fzowl committed:
Model alias & removing the  (#5)

* - adding alias field to ModelMeta
 - removing the prefix from sentence-transformer model
 - introducing E5 as subclss of sentence-transformer

* - adding alias field to ModelMeta
 - removing the prefix from sentence-transformer model
 - introducing E5 as subclss of sentence-transformer

* - adding alias field to ModelMeta
 - removing the prefix from sentence-transformer model
 - introducing E5 as subclss of sentence-transformer

fzowl committed:
Put the dataset metadata to results.json

fzowl committed:
Merge branch 'main' into e5-mistral

fzowl committed:
Put the dataset metadata to results.json
